### ISSUE TITLE
Confusing docs

### ISSUE THREAD

**Issue by paolomaffei** [typings/grunt-typings#1]
From the docs:

```
your_target: {
      // Target-specific file lists and/or options go here. 
    },
```

it doesn't say that you do need at least one target, and also I have no clue what the "target-specific file lists options" could be

**Comment by sebastianhaas** [comment]
True that, it's just the default documentation template of grunt-init. I didn't take the time to change it since there are no options anyway.

**Comment by sebastianhaas** [comment]
@paolomaffei I changed the docs and removed the unnecessary parts. Published to npm as 0.1.3.

If you need additional configuration options, I'm more than happy to accept PRs.
